admin/billing/biller.go

```diff
@@ -0,0 +1,101 @@
+package billing
+
+import (
+	"context"
+	"time"
+
+	"github.com/rilldata/rill/admin/database"
+)
+
+const (
+	DefaultPlanID   = "Starter"
+	SupportEmail    = "support@rilldata.com"
+	DefaultTimeZone = "UTC"
+)
+
+type Biller interface {
+	Name() string
+	GetDefaultPlan(ctx context.Context) (*Plan, error)
+	GetPlans(ctx context.Context) ([]*Plan, error)
+	// GetPublicPlans for listing purposes
+	GetPublicPlans(ctx context.Context) ([]*Plan, error)
+	// GetPlan returns the plan with the given Rill plan ID or biller plan ID.
+	GetPlan(ctx context.Context, rillPlanID string, billerPlanID string) (*Plan, error)
```

Comment:
If it's necessary to do lookups by both IDs, this should be broken into two functions – e.g. similar to how `database` has `FindOrganization(ctx, id)` and `FindOrganizationByName(ctx, name)`